src/linear_algebra/matrix/hermitian.lean

```diff
@@ -146,6 +150,16 @@ variables [ring α] [star_ring α] [ring β] [star_ring β]
 
 end ring
 
+section comm_ring
+
+variables [comm_ring α] [star_ring α]
+
+lemma is_hermitian_nonsingular_inv [fintype m] [decidable_eq m] {A : matrix m m α}
+  (hA : A.is_hermitian) : A⁻¹.is_hermitian :=
+by simp [is_hermitian, conj_transpose_nonsing_inv, hA.eq]
```

Comment:
```suggestion
lemma is_hermitian.inv [fintype m] [decidable_eq m] {A : matrix m m α}
  (hA : A.is_hermitian) : A⁻¹.is_hermitian :=
by simp [is_hermitian, conj_transpose_nonsing_inv, hA.eq]
```
It might also be nice to have `is_hermitian_inv : (⅟ A).is_hermitian ↔ A.is_hermitian` as well.

Reply:
I've added `(A⁻¹).is_hermitian ↔ A.is_hermitian`. Should I also add variants for `⅟A`?